doc/source/conf.py

```diff
@@ -1,31 +1,32 @@
-# -*- coding: utf-8 -*-
-import sys, os, re
-import glob
-from datetime import date
+import math
+import os
+from os.path import relpath, dirname
+import re
+import sys
 import warnings
+from datetime import date
 
-import numpy as np
-
-# Currently required to build scipy.fft docs
-os.environ['_SCIPY_BUILDING_DOC'] = 'True'
-
-# Check Sphinx version
-import sphinx
-if sphinx.__version__ < "2.0":
-    raise RuntimeError("Sphinx 2.0 or newer required")
```

Comment:
Why remove the version checks for ? Are these versions just too old to be likely (e.g. they only support Python 2 or something)?

Reply:
Yeah we have that in our requirements and Sphinx 4 lol. Having checks like that in the code is a bit pointless. We would want to be coherent, we would check all our deps all the time like that...